azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/inbox.py

```diff
@@ -0,0 +1,27 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module contains an Inbox class for use with a synchronous client."""
+
+from six.moves import queue
+
+
+class Inbox(queue.Queue):
```

Comment:
I agree that semantics are important.  I think this is a good temporary name, but I don't particularly like it.  (even though it was the one I used in that one meeting).  

update: I feel slightly better about the name if this object is entirely internal, and it looks like it might be (if I'm reading things correctly).

Reply:
It is. That was the impetus for the change - it's internal only, and MessageQueue won't generalize to Twins, etc.

---
In reply to: [262238256](https://github.com/Azure/azure-iot-sdk-python-preview/pull/55#discussion_r262238256) [](ancestors = 262238256)